tests/integrations/doc_index/elastic/fixture.py

```diff
@@ -1,6 +1,7 @@
 import os
```

Comment:
I think all tests ended up in the wrong directory, can you move them out of `integration` and into `index`?

Reply:
I think that was a mistake. Can you confirm @samsja?